bluesky/tests/test_new_examples.py

```diff
@@ -641,7 +641,9 @@ def test_no_rewind_device(hw):
 
 
 def test_monitor(RE, hw):
-    RE(monitor_during_wrapper(count([hw.det], 5), [hw.det1]))
+    from ophyd.sim import SynSignal
```

Comment:
Can this import be moved to the top of the file?

Reply:
I think this is intentional. Since bluesky does not have an ophyd dependency, we have laid some track for making it possible run the subset of the bluesky tests that don't require ophyd. The `hw` fixture is a way of declaring, "This test needs ophyd."

Reply:
Ah, I see. It makes sense even that we have it in https://github.com/bluesky/bluesky/blob/018d4bdd9e0f6d5365e75374f26247328763e1f6/dev-requirements.txt#L16